internal/cmd/cmd.go

```diff
@@ -220,8 +233,8 @@ func (c *Cmd) Start() error {
 	if c.Stdout != nil {
 		c.iogrp.Go(func() error {
 			_, err := relayIO(c.Stdout, stdout, c.Log, "stdout")
-			if err := p.CloseStdout(context.TODO()); err != nil {
-				c.Log.WithError(err).Warn("failed to close Cmd stdout")
+			if cErr := p.CloseStdout(context.TODO()); cErr != nil && !isClosedIOErr(cErr) && c.Log != nil {
```

Comment:
You don't need to use a different variable name here, it will already shadow `err` since it is a new scope.

Reply:
i changed it since it could cause confusion with `err` above if people didn't notice the `:=` vs `=`
(at least, it took me a while to notice it was overshadowing the `err` above 😅)

Reply:
Fair enough!